helix-view/src/handlers/dap.rs

```diff
@@ -321,6 +321,7 @@ impl Editor {
                             }
                         }
                         None => {
+                            self.debugger = None;
```

Comment:
This seems like a seperate bug fix to me. I think this is correct and needed to ensure that the debugger is actually removed once it terminates, right? This seems like the right fix to me and I ran into this bug before so it's great to see this fixed too

Reply:
Yeah it's separate, but related. Without this trying to add a breakpoint after the debugger terminated would show an error, better than crashing but still not great.

Reply:
yeah this is what I ran into too in the past, good catch :)